content/info/call-participation/shortpapers.md

```diff
@@ -0,0 +1,50 @@
+---
+title: shortpapers
+layout: page
+permalink: /info/call-participation/shortpapers
+contact: [committee-name]@ieeevis.org
+---
+
+---
+## IEEE Vis 2020 - Short Paper Call for Participation
+<!---
+layout: main-2020
+permalink: ???
+-->
```

Comment:
should remove this

Reply:
yes, but I can't find how :(
could you please point me to where to do that?

Reply:
I can remove it for you.  Normally, you can click the "Files Changed" tab at the top, click the "..." in the top-right corner by the filename, and click "Edit file."  This will allow you to edit the file, then you can commit an update.